_layouts/map.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+---
+<h2>{{ page.mapName }} v{{ page.version }}</h2>
+{% if page.mapCategory %}<h4>Category: {{ page.mapCategory }}</h4>{% endif %}
+{% if page.mapType == "MAP_SKIN" %}<h4>Type: Map Skin</h4>{% endif %}
+<img src="{{ page.img }}">
```

Comment:
I wonder if we should also have `page.img` inside an if block. 

Check markup counts in the maps yaml, we do have some data quality issues:
```
$ egrep -o "^  [a-z].*:" triplea_maps.yaml | sed 's/:.*//' | sort | uniq -c
    106   description
     93   img
     90   mapCategory
      1   mapName
     11   mapType
    105   url
    110   version
```
For some maps we will not have image it looks like. I think the options are to either always enforce an image, or put in tolerance for not having it. In the case of not having it, excluding altogether or using a default "no image" image would be some options.